modules/templates-writing-description.adoc

```diff
@@ -58,14 +58,107 @@ constants file.
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 The categories can also be customized for the whole cluster.
 endif::[]
-<6> An icon to be displayed with your template in the web console. Choose from
-our existing logo icons when possible. You can also use icons from FontAwesome.
-ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
-Alternatively, provide icons through CSS customizations that can be added to an
-{product-title} cluster that uses your template. You must specify an icon class
-that exists, or it will prevent falling back to the generic icon.
-endif::[]
-<7> The name of the person or organization providing the template.
+<6> An icon to be displayed with your template in the web console.
++
+.Available icons
+[%collapsible]
+====
+* `icon-3scale`
+* `icon-aerogear`
+* `icon-amq`
+* `icon-angularjs`
+* `icon-ansible`
+* `icon-apache`
+* `icon-beaker`
+* `icon-camel`
+* `icon-capedwarf`
+* `icon-cassandra`
+* `icon-catalog-icon`
+* `icon-clojure`
+* `icon-codeigniter`
+* `icon-cordova`
+* `icon-datagrid`
+* `icon-datavirt`
+* `icon-debian`
+* `icon-decisionserver`
+* `icon-django`
+* `icon-dotnet`
+* `icon-drupal`
+* `icon-eap`
+* `icon-elastic`
+* `icon-erlang`
+* `icon-fedora`
+* `icon-freebsd`
+* `icon-git`
+* `icon-github`
+* `icon-gitlab`
+* `icon-glassfish`
+* `icon-go-gopher`
+* `icon-golang`
+* `icon-grails`
+* `icon-hadoop`
+* `icon-haproxy`
+* `icon-helm`
+* `icon-infinispan`
+* `icon-jboss`
+* `icon-jenkins`
+* `icon-jetty`
+* `icon-joomla`
+* `icon-jruby`
+* `icon-js`
+* `icon-knative`
+* `icon-kubevirt`
+* `icon-laravel`
+* `icon-load-balancer`
+* `icon-mariadb`
+* `icon-mediawiki`
+* `icon-memcached`
+* `icon-mongodb`
+* `icon-mssql`
+* `icon-mysql-database`
+* `icon-nginx`
+* `icon-nodejs`
+* `icon-openjdk`
+* `icon-openliberty`
+* `icon-openshift`
+* `icon-openstack`
+* `icon-other-linux`
+* `icon-other-unknown`
+* `icon-perl`
+* `icon-phalcon`
+* `icon-php`
+* `icon-play`
+* `iconpostgresql`
+* `icon-processserver`
+* `icon-python`
+* `icon-quarkus`
+* `icon-rabbitmq`
+* `icon-rails`
+* `icon-redhat`
+* `icon-redis`
+* `icon-rh-integration`
+* `icon-rh-spring-boot`
+* `icon-rh-tomcat`
+* `icon-ruby`
+* `icon-scala`
+* `icon-serverlessfx`
+* `icon-shadowman`
+* `icon-spring-boot`
+* `icon-spring`
+* `icon-sso`
+* `icon-stackoverflow`
+* `icon-suse`
+* `icon-symfony`
+* `icon-tomcat`
+* `icon-ubuntu`
+* `icon-vertx`
+* `icon-wildfly`
+* `icon-windows`
+* `icon-wordpress`
+* `icon-xamarin`
+* `icon-zend`
+====
+<7> The name of the person or organization providing template.
```

Comment:
s/proving template/providing the template

Reply:
@codyhoag Great catch. Thanks!